app/auth/auth_service.ts

```diff
@@ -154,6 +154,7 @@ export class AuthService {
         )
       ),
       isImpersonating: Boolean(rpcService.requestContext.impersonatingGroupId),
+      subdomainGroupID: response.subdomainGroupId,
```

Comment:
nit: maybe this would fit better in the `config` proto since it's a property of the domain and not the user? It feels slightly error-prone to put this in the User object if the user is not necessarily authorized to access this group ID.

If we are worried about an extra DB request though in the backend, maybe this can be a field on the AuthService?

Reply:
It's related to the user: `subdomainGroupId` is only populated if the user is allowed to impersonate.